client/lib/plans/features-list.tsx

```diff
@@ -1782,24 +1779,6 @@ export const FEATURES_LIST: FeatureList = {
 		getAlternativeTitle: () => '2%',
 		getFeatureGroup: () => FEATURE_GROUP_PAYMENT_TRANSACTION_FEES,
 	},
-	[ FEATURE_50GB_STORAGE_ADD_ON ]: {
```

Comment:
**Note:**

Add ons are not features ( more discussed in pdvytD-iS-p2). We remove strorage add ons from the feature list and instead use existing add-on data definitions from `client/my-sites/add-ons/hooks/use-add-ons.ts`